test/PlaidClientTest.js

```diff
@@ -550,28 +350,77 @@ describe('plaid.Client', () => {
 
       describe('transactions', () => {
         let accessToken;
-        beforeEach(done => {
-          pCl.createItem({
-            username: testConstants.USERNAME,
-            password: testConstants.PASSWORDS.GOOD
-          }, testConstants.INSTITUTION, testConstants.PRODUCTS, {
-            transactions: {
-              start_date: now,
-              end_date: now,
-              await_results: true
+
+        var getTransactionsWithRetries = (accessToken, startDate, endDate,
```

Comment:
(nit) would be cool to use some generic retry function for both of these

Reply:
:zap: to the idea but going to punt this for the future